environment.yml

```diff
@@ -2,6 +2,7 @@ name: pubtator
 dependencies:
 - python=3.6.0
 - lxml=3.7.2
+- tqdm=4.11.2
+- pandas=0.20.2
 - pip:
-    - git+https://github.com/en-dash/PyBioC@a21bd181ce3b89de791db4bb7db6ca1c48167648
-    - tqdm==4.11.2
+    - git+https://github.com/en-dash/PyBioC@a21bd181ce3b89de791db4bb7db6ca1c48167648
```

Comment:
Add final newline. Note that all files tracked by git should end with a final newline. This helps with diffing the file going forward. See the red symbol github shows at the end of this line due to missing newline.